### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by hotzenklotz**
This PR updates the WK documentation:

- Revised the toolbar page for better readbility and include icons for each tool
- Updated User and Teams pages to include info on team creation and user assignment
- Updated Volume and Skeleton Tools pages to include icons for all tools and modifiers. Updated descriptions and phrasing.
- Misc formatting and link fixes  

Impressions: 
Screenshot 2024-12-09 at 14 44 58
Screenshot 2024-12-09 at 14 45 16

------
(Please delete unneeded items, merge only when none are left open)
- [ ] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [x] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [ ] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

File: docs/skeleton_annotation/tools.md [hunk 1]
```diff
@@ -1,33 +1,67 @@
...
 
-![Skeleton Tool modifiers](../images/skeleton_tool_modifiers.jpeg)
+### Branch Points
+- Mark any node as a branch point using ++b++ or the right-click menu
+- Branch points are highlighted with a distinct color
+- All branch points are stored in a first-in, first-out (FIFO) stack
+- Jump to the latest branch point using ++j++ to continue working from there
 
-## Controls & Keyboard Shortcuts for Skeleton Annotations
+### Tool Modifiers
```
Comment (philippotto) [inline docs/skeleton_annotation/tools.md:38]: hmm, I don't really like the word modifier here. how about "Tool-specific functionality"?
I only use the term modifier in the context of keyboard shortcuts (so, shift, ctrl etc. are modifiers). that's also how we use it in the UI afaik.
I suggest to simply omit modifier in the tool descriptions:
- "New Tree". maybe add "button" or sth similar if you think that's better. but it has nothing to do with a modifier from my understanding. 
- "Single Node Tree Mode" and "Merger Mode" <-- there is already the word mode in it. "modifier" doesnt improve it imo.

File: docs/skeleton_annotation/tools.md [hunk 2]
```diff
@@ -1,33 +1,67 @@
...
+![New Tree Modifier](./images/new-tree-modifier.jpg){align=left width="60"} 
+**Create new Tree**
+
+- Starts a fresh tree structure
+- Useful when annotating multiple separate structures
+
+![Single Node Tree Mode Modifier](./images/single-node-tree-mode-modifier.jpg){align=left width="60"} 
+**Single Node Tree Mode**
+
+- Creates a new tree for each placed node
```
Comment (philippotto) [inline docs/skeleton_annotation/tools.md:51]:
```suggestion
- When enabled, creates a new tree for each placed node
```

to make it clearer that it is a mode

File: docs/ui/toolbar.md [hunk 3]
```diff
@@ -1,41 +1,107 @@
 # The Toolbar
 
-The toolbar contains frequently used commands, such as saving and sharing, your current position within the dataset, and the ability to switch between various modes for viewing. Further, it provides access to all the tools for annotation, navigation, and more.
+The toolbar is your primary interface for interacting with WEBKNOSSOS. Located at the top of the screen, it provides quick access to essential functions for viewing, annotating, and sharing your work. This guide explains each component and its purpose in your workflow.
```
Comment (philippotto) [inline docs/ui/toolbar.md:3]: might be nitpicky, but for me the primary interface are the data viewports + mouse :see_no_evil: but I dont want to be päbstlicher als der pabst.

File: docs/volume_annotation/tools.md [hunk 4]
```diff
@@ -1,64 +1,128 @@
...
+  - AI-based segmentation (powered by Segment Anything Model 2)
+    The AI mode works across various imaging modalities and can significantly speed up annotation workflows. See the [Quick-select tool](#quick-select-tool) section for detailed usage.
+
+![Proof Reading Tool](../ui/images/proofreading-tool.jpg){align=left width="60"} 
+**Proof Reading**: Fix merge and split errors in automated segmentations using the underlying super-voxel graph by combining and breaking apart segments. Read more about [proofreading](../proofreading/tools.md).
+
+
+In the [Segments tab](./segments_list.md) on the right-hand side panel, you can find all segment IDs which are available in your annotation. You can rename and organize segments as needed.
+
+The active segment ID under the cursor can be found in the [status bar](../ui/status_bar.md) at the bottom of the screen or through the context-sensitive menu when right-clicking.
```
Comment (philippotto) [inline docs/volume_annotation/tools.md:42]: more concise and well established

```suggestion
The active segment ID under the cursor can be found in the [status bar](../ui/status_bar.md) at the bottom of the screen or through the context menu when right-clicking.
```

File: docs/volume_annotation/tools.md [hunk 5]
```diff
@@ -1,64 +1,128 @@
...
+
+!!! tip "Keyboard Shortcuts"
+    For faster workflows, refer to the [keyboard shortcuts](../ui/keyboard_shortcuts.md) guide.
+
+
+### Tool Modifiers
```
Comment (philippotto) [inline docs/volume_annotation/tools.md:48]: same as with the skeleton tools

File: docs/volume_annotation/tools.md [hunk 6]
```diff
@@ -1,64 +1,128 @@
...
+The following interactions and modifiers become available when working with some of the volume annotation tools. They can be found in the toolbar:
+
+![Create New Segment ID Modifier](./images/new-segment-modifier.jpg){align=left width="60"} 
+**Create New Segment ID**: Creates a new segment ID for labeling with a different color and identifier. New segments will be added to the [segments list](./segments_list.md) in the right sidebar automatically.
 
-![youtube-video](https://www.youtube.com/embed/JkpSTKuNZKg)
+![Change Brush Size Modifier](./images/brush-size-modifier.jpg){align=left width="60"} 
+**Change Brush Size**: Changes the size and radius of the brushing tool. Presets of small, medium and large brush sizes are available which can be modified to suite your workflows. ![youtube-video](https://www.youtube.com/embed/JkpSTKuNZKg)
```
Comment (philippotto) [inline docs/volume_annotation/tools.md:55]:
```suggestion
**Change Brush Size**: Changes the size and radius of the brushing tool. Presets of small, medium and large brush sizes are available which can be modified to suit your workflows. ![youtube-video](https://www.youtube.com/embed/JkpSTKuNZKg)
```

File: docs/volume_annotation/tools.md [hunk 7]
```diff
@@ -1,64 +1,128 @@
...
 
-In the `Segmentation` tab on the right-hand side panel, you can see the segment IDs which are available in your annotation. You can rename segments as needed.
+![Overwrite Everything Modifier](./images/overwrite-everything-modifier.jpg){align=left width="60"} 
+**Overwrite Everything**: When labeling with the brush/trace tool, you can annotate every available voxel without any regard if it was already labeled as a different segment or whether it is unlabelled. This allows you to draw over existing segments.
```
Comment (philippotto) [inline docs/volume_annotation/tools.md:58]:
```suggestion
**Overwrite Everything**: When labeling with the brush/trace tool, you can annotate every voxel independent of whether it is already an segment id. This allows you to draw over existing segments.
```

File: docs/volume_annotation/tools.md [hunk 8]
```diff
@@ -1,64 +1,128 @@
...
+- Disable the "AI" button
+- Uses intensity-based segmentation
+- Fills from the center of your selection
+
+### Usage Steps
+
+1. Select the Quick Select tool from the toolbar
+2. Choose your preferred mode (AI or threshold)
+3. Click the settings icon to configure:
+    - Number of sections to process
```
Comment (philippotto) [inline docs/volume_annotation/tools.md:93]: I think this is only possible for the AI mode.

File: docs/volume_annotation/tools.md [hunk 9]
```diff
@@ -1,64 +1,128 @@
...
 
 ## Quick-select tool
-The built-in quick select tools allows you draw a selection around a cell or object and WEBKNOSSOS will use machine-learning to automatically do the segmentation for you. The feature is based on the [Segment Anything Model 2](https://ai.meta.com/blog/segment-anything-2/) and works across a wide range of imaging modalities.
+The Quick Select tool offers AI-powered automatic segmentation, powered by [Segment Anything Model 2](https://ai.meta.com/blog/segment-anything-2/). Simply draw a selection around your target structure, and WEBKNOSSOS will automatically segment it for you.
+
+### Operating Modes
+
+**AI Mode** (Default)
+
+- Activate the "AI" button in the toolbar
```
Comment (philippotto) [inline docs/volume_annotation/tools.md:78]: probably nitpicky, but it's a bit weird that the headline says that it's enabled by default and then the description contains the imperative "activate it".
maybe move the part about activating/deactivating directly into the intro of "Operating Modes".

File: docs/volume_annotation/tools.md [hunk 10]
```diff
@@ -1,64 +1,128 @@
...
+- Uses intensity-based segmentation
+- Fills from the center of your selection
+
+### Usage Steps
+
+1. Select the Quick Select tool from the toolbar
+2. Choose your preferred mode (AI or threshold)
+3. Click the settings icon to configure:
+    - Number of sections to process
+    - Preview mode for real-time parameter adjustment
```
Comment (philippotto) [inline docs/volume_annotation/tools.md:94]: only for non-ai

**Review by philippotto — COMMENTED** [review]
awesome, thanks for putting work into updating the docs :+1: I really like it. I left various suggestions, but several of them go into nitpick territory. so, feel free to dismiss comments.

File: docs/skeleton_annotation/tools.md [hunk 11]
```diff
@@ -1,33 +1,67 @@
...
 
-When the `Skeleton` tool is active, the following modifiers become available:
+## Creating Annotations
 
-- `Create new Tree`: Creates a new tree.
-- `Toggle single node tree mode`: This modifier makes the skeleton annotation tool create a new tree for each node instead of adding nodes to the current tree. You can use this mode to mark single objects or seeds, such as nuclei. This is also known as "Soma-clicking mode".
-- `Toggle merger mode`: This modifier activates the `Merger Mode` for the skeleton annotation tool. In merger mode, you can use skeletons to "collect" and merge volume segments from an over-segmentation. [Read more about `Merger Mode`](../proofreading/merger_mode.md).
+### New Trees
+- Select the **Skeleton** tool from the toolbar
+- Create a new tree using the toolbar button or press ++c++
```
[the remark this answers is not in the document]
Comment (hotzenklotz) [reply docs/skeleton_annotation/tools.md:28]: Yes, it formats the key as a keyboard key with CSS: https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#keys

File: docs/users/teams.md [hunk 12]
```diff
@@ -7,3 +7,18 @@ All new organization members are assigned to that team by default.
 An organization can contain as many teams as you like. Users can be assigned to multiple teams.
 
 ![All users, annotations, and datasets belong to an organization. All users are assigned to a default team. Further teams can be created for fine-grained access permissions.](../images/teams.jpeg)
+
+## Creating New Teams
+To create a new team for your organization:
+
+1. Navigate to `Admin > Teams` in the navigation bar
+2. Click the `Add Team` button in the upper right corner
+3. In the modal that appears:
```
[the remark this answers is not in the document]
Comment (hotzenklotz) [reply docs/users/teams.md:16]: Good idea

File: docs/ui/toolbar.md (hunk 3 above)
Comment (hotzenklotz) [reply docs/ui/toolbar.md:3]: :+1:

File: docs/volume_annotation/tools.md [hunk 13]
```diff
@@ -1,64 +1,129 @@
...
+
+![Segment Picker Tool](../ui/images/segment-picker-tool.jpg){align=left width="60"} 
+**Segment Picker**: Click any segment to use its label ID as the active segment ID and keep annotating with that ID. This is alternative to selecting the segment ID from the [Segments list](./segments_list.md) sidebar or context menu.
+
+![Quick Select Tool](../ui/images/quickselect-tool.jpg){align=left width="60"} 
+**Quick Select**: Automatically annotate segments using either:
+
+  - Threshold-based selection mode
+  - AI-based segmentation (powered by Segment Anything Model 2)
+    The AI mode works across various imaging modalities and can significantly speed up annotation workflows. See the [Quick-select tool](#quick-select-tool) section for detailed usage.
```
[the remark this answers is not in the document]
Comment (hotzenklotz) [reply docs/volume_annotation/tools.md:35]: This is incorrect. The SAM2 model predicts on several "layers" at once. It is capaple of understanding images sequences.

**Pull request merged**